sql/src/main/java/io/crate/execution/dml/upsert/CheckConstraints.java

```diff
@@ -73,6 +86,22 @@ public void validate(T values) {
                 throw new IllegalArgumentException("\"" + notNullColumns.get(i) + "\" must not be null");
             }
         }
+        for (Map.Entry<String, Tuple<Input<?>, String>> checkEntry : checkConstraints.entrySet()) {
+            Input<?> checkInput = checkEntry.getValue().v1();
+            Boolean value = (Boolean) checkInput.value();
+            if (null == value) {
+                continue;
+            }
```

Comment:
Is that a valid behavior?

Reply:
I interpreted it so: 

```
It should be noted that a check constraint is satisfied if the check expression evaluates to true or the null value. Since most expressions will evaluate to the null value if any operand is null, they will not prevent null values in the constrained columns. To ensure that a column does not contain null values, the not-null constraint described in the next section can be used.
```

https://www.postgresql.org/docs/9.4/ddl-constraints.html

Reply:
Could you add short inline comment about this?

In other cases `null` evaluates to `false`, so this behavior here might look like a bug on the first glance.